boto/auth.py

```diff
@@ -51,23 +51,27 @@
     sha256 = None
 
 
-# Region detection strings to determine if SigV4 should be used
-# by default.
-SIGV4_DETECT = [
-    '.cn-',
-    # In eu-central and ap-northeast-2 we support both host styles for S3
-    '.eu-central',
-    '-eu-central',
-    '.ap-northeast-2',
-    '-ap-northeast-2',
-    '.ap-south-1',
-    '-ap-south-1',
-    '.us-east-2',
-    '-us-east-2',
-    '-ca-central',
-    '.ca-central',
-    '.eu-west-2',
-    '-eu-west-2',
+# Region detection strings to determine if SigV2 should be used
+# by default
+SIGV2_DETECT = [
```

Comment:
Can we rename this to `S3_AUTH_DETECT` or something along those lines?  This is specifically for the older s3 signature version (which is not sigv2) and we should use consistent naming in boto2, which I believe calls it either `s3` or `hmac-v1`.